client/actions/types.js

```diff
@@ -0,0 +1,33 @@
+export const ADD_FLASH_MESSAGE = 'ADD_FLASH_MESSAGE';
```

Comment:
While this is not too hard to manage right now seeing as only a few types have been defined, it will definitely become harder to manage in the future when you need to add more action types. 

It will be better if you break everything further down into sub folders holding related functionality; for instance, everything related to authentication can be kept in a folder called `auth` and within said folder, action types specific to authentication can be kept.